hardware/P1MeterBase.h

```diff
@@ -25,9 +25,9 @@ class P1MeterBase : public CDomoticzHardwareBase
 	unsigned char m_exclmarkfound;
 	unsigned char m_CRfound;
 
-	float m_voltagel1;
-	float m_voltagel2;
-	float m_voltagel3;
+	float m_p1voltagel1;
```

Comment:
Maybe we could not rename this variables as we are already in the P1 class... thats double,
but maybe it's an idea to rename these two:

	P1Power	m_p1power;
	P1Gas	m_p1gas;